ops/testing.py

```diff
@@ -11,8 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Framework for unit testing charms in a simulated environment.
```

Comment:
"simulated Juju* environment" might be clearer

Reply:
done